src/components/DataViewer/NodeParentKey.tsx

```diff
@@ -60,7 +60,9 @@ function renderBreadcrumbs(
   navigateToRef: (ref: firebase.database.Reference) => void
 ) {
   const ancestors = getAncestorRefs(activeRef);
-  const crumbs = ancestors.reduce<JSX.Element[]>((crumbs, ref, i) => {
+
+  let crumbs: JSX.Element[] = [];
+  ancestors.forEach((ref, i) => {
```

Comment:
nit: This can probably use `for-of` loops. If you do feel strongly about `forEach`, please remove the second argument (the `[]` in `}, []);` on Line 81).